docs/pages/management/join-services-to-your-cluster/join-token.mdx

```diff
@@ -0,0 +1,328 @@
+---
+title: Join Services with a Secure Token
+description: How to join services with your Teleport cluster via a join token
+---
+
+In this guide, we will show you how to register a Teleport process running one
+or more services to your cluster by presenting a **join token**. 
+
+In this approach, you declare your intention to register a new Teleport process,
+and Teleport generates a secure token that the process uses to establish a trust
+relationship with the Teleport cluster.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A Linux server that you will use to host your Teleport process, e.g., a
+  virtual machine or Docker container with an image based on a Linux
+  distribution.
+
+  In this guide, we will show you how to register a Teleport SSH Service
+  instance. This approach applies to new Proxy Service instances as well as the
+  Kubernetes Service, Database Service, and other Teleport services for
+  accessing resources in your infrastructure.
+
+  <Details type="tip" title="Running multiple Proxy Service instances behind a
+  load balancer" >
+
+  The join token method works if a cluster includes a single Proxy Service
+  instance as well as multiple Proxy Service instances behind a load balancer
+  (LB) or a DNS entry with multiple values. If there are multiple Proxy Service
+  instances, a Teleport process joining the cluster establishes a tunnel to
+  every Proxy Service instance.
+  
+  If you are using a load balancer, it must use a round-robin or a similar
+  balancing algorithm. Do not use sticky load balancing algorithms (i.e.,
+  "session affinity") with Teleport Proxy Service instances.  
+
+  </Details>
+
+  <Notice type="tip" >
+
+  If you are are using a Docker container, note that this guide assumes that
+  your Linux host has `curl` and `sudo` installed. 
+
+  </Notice>
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/3. Install Teleport
+
+Install Teleport on your Linux host.
+
+(!docs/pages/includes/install-linux.mdx!)
+
+## Step 2/3. Join your Teleport process to the cluster
+
+In this section, we will join your Teleport process to your cluster by:
+
+- Obtaining a join token
+- Running your Teleport process with the join token
+
+### Generate a token
+
+Teleport only allows access to resources in your infrastructure via Teleport
+processes that that have joined the cluster.
+
+On your local machine, use the `tctl` tool to generate a new token. In the
+following example, a new token is created with a TTL of five minutes:
+
+```code
+# Generate a short-lived invite token for a new Teleport SSH Service instance:
+$ tctl tokens add --ttl=5m --type=node 
+The invite token: (=presets.tokens.first=)
+This token will expire in 5 minutes.
+
+Run this on the new node to join the cluster:
+
```

Comment:
While we're here, can we replace this with `teleport node configure` + `teleport start`?

Reply:
This is the output of the `tctl tokens add` command, which [still prints](https://github.com/gravitational/teleport/blob/8a18b2b58ed5baaf4e5f30f23c948122d3a1d9a7/tool/tctl/common/node_command.go#L119-L134) the `teleport start` command

Reply:
We _could_ be more explicit in saying that we only care about the token and will ignore the rest of the output. We could also `grep` the output for the token.

Or we could update `tctl` to provide the desired output.